src/components/ParamsInput/ParamsInput.js

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { TextInput } from 'carbon-components-react';
+import TextInputSkeleton from 'carbon-components-react/lib/components/TextInput/TextInput.Skeleton';
+
+const ParamsInput = ({
+  name,
+  description,
+  defaultValue,
+  loading,
+  ...textInputProps
+}) => {
+  if (loading) {
+    return <TextInputSkeleton />;
+  }
+  return (
+    <TextInput
+      labelText={name}
+      helperText={description}
+      defaultValue={defaultValue}
+      placeholder={defaultValue}
```

Comment:
we should only set one of `defaultValue` or `placeholder`
I'd remove both of these here and leave it up to the consumer

similar with labelText and helperText, let the consumer set them directly so we have an API consistent with the TextInput, the only addition is the `loading` prop.

Reply:
Ok. Should I rename the component to something more generic as well?

Reply:
Like `LoadingTextInput`? Also, is it worth creating a component for this?

Reply:
`TextInput` is fine, and yes I think it's worth creating a component for this since it's likely to be reused quite a bit and will ensure consistency.

Reply:
Ok, thanks. I'll make the update